docs/investigations/file-upload-with-graphql.md

```diff
@@ -0,0 +1,121 @@
+# File upload with graphql
+
+## Overview
+
+This document discuss the following:
```

Comment:
I wouldn't use points in the overview. Please remove them and instead write the following:
```suggestion
This document discusses how to upload files using graphql and suggests how to handle large data objects.
```